### ISSUE TITLE
TSR information

### ISSUE THREAD

**Issue by christianstahl** [openETCS/modeling#891]
Dear Jakob (@JakobGartner),

can the track atlas provide the following information:
"inhibition of revocable TSRs from balises in L2/3” is stored on-board" ?
This condition needs to be evaluated in the information fiter.

Thanks,
Christian

**Comment by JakobGartner** [comment]
yes, it will be available from the next release tomorrow (I hope tomorrow)

however, on our track we only see non- revocable TSRs from RBC, so I guess you can hardcode it for now.
